tests/Struct/FindologicProductTest.php

```diff
@@ -390,13 +396,33 @@ public function testProduct(): void
             $config
         );
 
+        $productKeyword = new Keyword('FINDOLOGIC Keyword');
+        $ignoreKeyords = [
+            $productEntity->get('id'),
```

Comment:
Are we getting the product ID as well in the keywords? I thought it was just manufacturer name and product number?